modules/recon/hosts-hosts/ipinfodb.py

```diff
@@ -27,9 +27,15 @@ def module_run(self, hosts):
                 self.error(jsonobj['statusMessage'])
                 continue
             time.sleep(.7)
```

Comment:
I just noticed this. Did you see this before?

Reply:
I did I suppose that time could just be increased. Although I still think we should just make it variable since if someone purchases the API they do not hit these same 2 Queries per second restriction.

Reply:
I agree. That's fine. But rather than do the count thing, just adjust the sleep input accordingly. Also, remove the other sleep call? Seems like there should only be one.

Reply:
pushed

Reply:
I think you misunderstood me. I meant I agreed on making it flexible so that those with paid API access don't have a rate limit. Also, 2 requests per second would be a 0.5 sleep, not a 1.5 right? So sleep should be `options['limit']: sleep(0.5)`, where `limit` is a boolean option for whether their requests should be rate limited or not. That sound right?

Reply:
This should be good now.